gluten-core/src/main/scala/io/glutenproject/expression/ConverterUtils.scala

```diff
@@ -513,6 +515,10 @@ object ConverterUtils extends Logging {
 
   // SparkSQL DateTime functions of Velox
   final val DAY_OF_MONTH = "day_of_month"
+  final val FROM_UNIXTIME = "from_unixtime"
+  final val DATE_ADD = "date_add"
```

Comment:
We can align the name with substrait's for some of these functions. Right?
https://github.com/substrait-io/substrait/blob/7f272f13f22cd5f5842baea42bcf7961e6251881/extensions/functions_datetime.yaml#L37